### DISCUSSION TITLE
Magic Search on Mobile

### DISCUSSION THREAD

**Discussion by EntropicNinja** [ente-io/ente#3338] [Enhancements]
I have Ente in my phone but not on a desktop.

I enabled Magic Search and it works well, but think maybe the reason my battery life has taken a nose dive over the last couple of days as my photo capturing has increased.

I just turned it off to test, but couldn't use magic search any longer.

My request here is to somehow decouple magic search from continious indexing. The ability to enable Machine Learning to index, then disable it to conserve battery, but still be able to search for items, etc. would be a boon!

Alternatively, maybe even the ability to schedule when on charge and/or overnight.

I know it doesn't delete the database, as I can turn it back on and it does not need to index my catalogue again to enable magic search!

This begs the question, if I leave a desktop to index, how does one use magic search on mobile? 🤔

**Comment by ua741** [comment]
This is a valid feedback, and feature request. We will prioritize and ship the same in the future. Hopefully, before end of Q1 2025.

**Comment by EntropicNinja** [comment]
Many thanks for implementing this recently.

I'd like to give feedback, if I may.

I can not enable local indexing without first enabling machine learning. Therefore, I can not check that its enabled without ticking "I understand, and wish to enable machine learning".

Perhaps show this page only if toggling the configuration to " Enable".

I checked "More Details" for some information and it appears that you have not finished the write up of Local Indexing as the sentence stops at "...either." See image

![ResizedImage_2024-11-10_09-42-48_1172](https://github.com/user-attachments/assets/3b5da49b-938c-448e-93c6-ba038866b5fa)

**Comment by NeverRIEght** [comment]
Close this as implemented, please :)
